pkg/gofr/metrics_server.go

```diff
@@ -11,24 +12,34 @@ import (
 
 type metricServer struct {
 	port int
+	srvr *http.Server
```

Comment:
`srv` is used in other file, not srvr

```suggestion
	srv *http.Server
```

Reply:
Sure changed back to srv